client/src/components/aggregations/utils.js

```diff
@@ -188,3 +189,34 @@ export const likertScaleAndPieAggregation = (fieldName, fieldConfig, data) => {
     }
   }
 }
+
+export const CALENDAR_OBJECT_TYPES = {
+  Report: "Report"
```

Comment:
In keeping with the convention in other places, I prefer:
```
  REPORT: "Report"
```

Reply:
Done